crates/iota-sdk-ffi/src/lib.rs

```diff
@@ -137,3 +138,24 @@ pub fn hex_encode(input: &[u8]) -> String {
 pub fn hex_decode(input: String) -> crate::error::Result<Vec<u8>> {
     Ok(hex::decode(input)?)
 }
+
+macro_rules! export_basic_types_bcs_conversion {
```

Comment:
Can we call this `export_primitive_types_bcs_conversion`

Reply:
Yeah I'd say primitive as well